### PR TITLE
added spring boot debug logging

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

**Comment by rhoads-zach** [comment]
@Ladicek @metacosm mind taking a look?

**Comment by metacosm** [comment]
@rhoads-zach will do

File: docs/topics/proc_add-debug-logging-spring-boot.adoc [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+public Greeting greeting(@QueryParam("name") @DefaultValue("World") String name) {
+    String message = String.format(properties.getMessage(), name);
+    
+    logger.debug("Message: " + message);
+    
+    return new Greeting(message);
+}
+----
+
+. Add a `logging.level.path.to.MY_CLASS=DEBUG` in `src/resources/application.properties`.
```
Comment (Ladicek) [inline docs/topics/proc_add-debug-logging-spring-boot.adoc]: Instead of `path.to.MY_CLASS`, I'd use `fully.qualified.name.of.TheClass`.

**Comment by Ladicek** [comment]
LGTM otherwise.

**Comment by rhoads-zach** [comment]
thank you @Ladicek for the feedback.

@metacosm any other feedback or am i free to merge?

File: docs/topics/proc_add-debug-logging-spring-boot.adoc [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+public Greeting greeting(@QueryParam("name") @DefaultValue("World") String name) {
+    String message = String.format(properties.getMessage(), name);
+    
+    logger.debug("Message: " + message);
+    
+    return new Greeting(message);
+}
+----
+
+. Add a `logging.level.full.qualified.name.of.TheClass=DEBUG` in `src/resources/application.properties`.
```
Comment (Ladicek) [inline docs/topics/proc_add-debug-logging-spring-boot.adoc]: It's not "full", it's "fully". _Fully qualified name_ is actually a technical term: https://en.wikipedia.org/wiki/Fully_qualified_name

File: docs/topics/proc_show-debug-logging-local-spring-boot.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+----
+
+. View your application logs to see your debug messages.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+i.o.booster.service.GreetingEndpoint     : Message: Hello, Sarah!
+----
+
+To disable debug logging, remove `logging.level.full.qualified.name.of.TheClass=DEBUG` from `src/resources/application.properties` and restart your application.
```
Comment (Ladicek) [inline docs/topics/proc_show-debug-logging-local-spring-boot.adoc]: Same here, "fully".

**Comment by rhoads-zach** [comment]
@Ladicek nice catch, sorry for missing this. Fixed

**Comment by metacosm** [comment]
Will review it shortly.

**Comment by metacosm** [comment]
LGTM.

**Comment by rhoads-zach** [comment]
thank you @metacosm merging.

**Pull request merged**